tutorial/westeros/westeros.ipynb

```diff
@@ -0,0 +1,1268 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "beta = 0.7 # an elasticity of demand\n",
+    "gdp = pd.Series([1., 1.21631, 1.4108], index=pd.Index(horizon, name='Time'))\n",
+    "demand_profile = gdp ** beta\n",
+    "demand_profile.plot(title='Demand')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The `COMMODITY_BALANCE` equation ensures that `input`, `output`, and `demand` are met at each `level` in the energy system\n",
+    "\n",
+    "$\\sum_{\\substack{n^L,t,m \\\\ y^V \\leq y}} output_{n^L,t,y^V,y,m,n,c,l}\n",
+    "     \\cdot ACT_{n^L,t,y^V,y,m}$  \n",
+    "     \n",
+    "$- \\sum_{\\substack{n^L,t,m, \\\\ y^V \\leq y}} input_{n^L,t,y^V,y,m,n,c,l}\n",
+    "     \\cdot ACT_{n^L,t,m,y} $  \n",
+    "\n",
+    "$- demand_{n,c,l,y} \\geq 0 \\quad \\forall \\ l \\in L$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "First we establish demand. Let's assume\n",
+    "\n",
+    "- 40 million people in [300 AC](https://atlasoficeandfireblog.wordpress.com/2016/03/06/the-population-of-the-seven-kingdoms/)\n",
+    "- similar population growth to Earth in the same time frame [(~factor of 12)](https://en.wikipedia.org/wiki/World_population_estimates)\n",
+    "- similar per capita demand for lighting as a developed country today (e.g., ~6000 GWh in Austria)\n",
```

Comment:
could include a reference here to the public IEA energy statistics at http://www.iea.org/statistics/statisticssearch/report/?country=AUSTRIA&product=electricityandheat&year=2015 to give users an easy starting point to adapt to other countries

Reply:
done!